Author: Roland Hieber <rohieb@rohieb.name>
Subject: Build HTML documentation suited for offline usage

Currently, the generated documentation includes a search box which relies on an
external search service, and which does not work for local usage. Therefore, the
generated search box does not work and confuses users, so it is best to build
the documentation in offline format which does not include the search box, but
does also not confuse users.

Index: qt4-x11-4.8.5+dfsg/doc/doc.pri
===================================================================
--- qt4-x11-4.8.5+dfsg.orig/doc/doc.pri	2013-06-07 07:16:49.000000000 +0200
+++ qt4-x11-4.8.5+dfsg/doc/doc.pri	2013-08-26 16:49:44.000000000 +0200
@@ -35,7 +35,7 @@
     EXAMPLESMANIFESTTARGET = $$replace(EXAMPLESMANIFESTTARGET,  "/", "\\")
     DEMOSMANIFESTTARGET = $$replace(DEMOSMANIFESTTARGET,  "/", "\\")
 }
-ADP_DOCS_QDOCCONF_FILE = qt-build-docs-online.qdocconf
+ADP_DOCS_QDOCCONF_FILE = qt-build-docs.qdocconf
 DITA_DOCS_QDOCCONF_FILE = qt-ditaxml.qdocconf
 QT_DOCUMENTATION = ($$QDOC qt-api-only.qdocconf assistant.qdocconf designer.qdocconf \
                     linguist.qdocconf qmake.qdocconf qdeclarative.qdocconf) && \
